backend/server/schema.py

```diff
@@ -28,20 +36,130 @@ class Meta:
 
 
 class PredictionType(DjangoObjectType):
+    """Basic prediction type based on Prediction data model"""
+
     class Meta:
         model = Prediction
 
-    is_correct = graphene.Boolean()
 
+class ModelPredictionType(graphene.ObjectType):
+    """
+    Stats for predictions made by the given model through the given round
+    """
+
+    model_name = graphene.String()
+    cumulative_correct_count = graphene.Int()
+
+    def resolve_model_name(self, _info):
+        return self.get("model_name")
+
+    def resolve_cumulative_correct_count(self, _info):
+        return self.get("cumulative_correct_count")
+
+
+class RoundPredictionType(graphene.ObjectType):
+    """Predictions for the year grouped by round"""
+
+    round_number = graphene.Int()
+    model_predictions = graphene.List(
+        ModelPredictionType,
+        description=(
+            "Stats for predictions made by the given model through the given round"
+        ),
+    )
+
+    def resolve_round_number(self, _info):
+        return self.get("match__round_number")
+
+    def resolve_model_predictions(self, _info):
+        return [
+            {"model_name": model_name, "cumulative_correct_count": correct_count}
+            for model_name, correct_count in self.items()
+            if model_name != "match__round_number"
```

Comment:
is this double underscore a convention of the model?

Reply:
Yeah, that's how Django ORM handles nested models/objects, so instead of something like `Prediction.where(match: { round_number: 5 })`, which is how it would be in Rails, it's `Prediction.objects.filter(match__round_number=5)`.